search-commons/src/test/java/no/unit/nva/search2/ResourceQueryTest.java

```diff
@@ -143,6 +143,10 @@ static Stream<URI> uriProvider() {
             URI.create("https://example.com/"),
             URI.create("https://example.com/?fields=category,title,created_date"),
             URI.create("https://example.com/?query=Muhammad+Yahya&fields=CONTRIBUTOR"),
+            URI.create("https://example.com/?CONTRIBUTOR=https://api.dev.nva.aws.unit.no/cristin/person/1136254"),
+            URI.create("https://example.com/?CONTRIBUTOR_SHOULD="
+                       + "https://api.dev.nva.aws.unit.no/cristin/person/1136254+"
```

Comment:
are you using `+ `as a delimiter for multivalued parameters? Should I use it as well for facets? We should discuss

Reply:
actually it is an encoded space.